docs/UserGuide.md

```diff
@@ -395,7 +395,9 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: There are 2 ways to transfer data.
+1. If you want to use the `.json` file generated by BizBook, install the app in another computer and overwrite the data folder located at `[JAR file location]/data` with the file that contains the data from your previous BizBook home folder.
+2. If you want to import using a `.vcf` file, export the data using the `export` command in BizBook, i.e. `export f/vcf`. The `vcf` file will be located in a folder at `[JAR file location]/export/bizbook.vcf`. You can transfer this file to other devices using BizBook and to import the data use the `import` command, i.e. `import f/vcf p/[Path to the VCF file]/<file name>.vcf`.
```

Comment:
Could you add a comment to warn the user that user preferences will not be transferred?